README.md

```diff
@@ -0,0 +1,144 @@
+# Milvus Helm Chart
+
+* For more information about installing and using Helm, see the [Helm Docs](https://helm.sh/docs/). For a quick introduction to Charts, see the [Chart Guide](https://helm.sh/docs/topics/charts/).
+
+* Installs the milvus system [Milvus installation](https://milvus.io/docs/guides/get_started/install_milvus/install_milvus.md)
+
+## Introduction
+This chart bootstraps an milvus deployment on a Kubernetes cluster using the Helm package manager.
```

Comment:
This chart bootstraps Milvus deployment on a .....

Reply:
Done